emotes.go

```diff
@@ -37,10 +22,10 @@ func loadEmotes() error {
 	return nil
 }
 
-func processEmoteDir(path string) (common.EmotesMap, error) {
-	dirInfo, err := ioutil.ReadDir(path)
+func processEmoteDir(dir string) (common.EmotesMap, error) {
+	dirInfo, err := os.ReadDir(dir)
 	if err != nil {
-		return nil, fmt.Errorf("could not open emoteDir: %w", err)
+		common.LogErrorf("could not open emote dir: %v\n", err)
```

Comment:
return early might be useful here

Reply:
You bring up a good point. I got rid of the return because the emotes map was needed, BUT we could just return back a default emotes map